File: CrushingPlant/CrushingPlantApi/Models/Equipment.cs
namespace CrushingPlantApi.Models
{
    public class Equipment
    {
        public string Id { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string Status { get; set; } = string.Empty;
        public string Type { get; set; } = string.Empty;
        public DateTime UpdatedAt { get; set; }
    }
}

File: Client/CrushingPlant.Client/src/app/_models/equipment.ts
export class Equipment {
  id: string;
  name: string;
  status: string;
  type: string;
  updatedAt: Date;

  constructor(
    id: string,
    name: string,
    status: string,
    type: string,
    updatedAt: Date,
  ) {
    ((this.id = id),
      (this.name = name),
      (this.status = status),
      (this.type = type));
    this.updatedAt = updatedAt;
  }
}
